openassessment/xblock/static/js/fixtures/templates.json

```diff
@@ -682,8 +684,52 @@
         "output": "oa_staff_area.html"
     },
     {
-        "template": "openassessmentblock/staff_area/student_info.html",
+        "template": "openassessmentblock/staff_area/oa_student_info.html",
```

Comment:
I'm confused-- there is another block below for oa_student_info.html (line 761). Are there supposed to be 2? If so, how does this work?